deploy/README.md

```diff
@@ -156,10 +149,6 @@ helm install ./deploy/helm/individual/partsunlimitedmrp --name=client --set imag
 ### Backend - Order Service
 
 ```bash
```

Comment:
Why is the readme still referencing helm charts that are deleted on this same PR? For example line 158 references /individual/orderservice which is deleted.